Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -110,11 +110,20 @@ def setup_test_formula(name, content = nil)
     formula_path = CoreTap.new.formula_dir/"#{name}.rb"
 
     case name
-    when "testball"
+    when "foo"
+      content = <<-EOS.undent
+        url "https://example.com/#{name}-1.0"
+      EOS
+    when "bar"
+      content = <<-EOS.undent
+        url "https://example.com/#{name}-1.0"
+        depends_on "foo"
+      EOS
+    else
       content = <<-EOS.undent
         desc "Some test"
         homepage "https://example.com/#{name}"
-        url "file://#{File.expand_path("..", __FILE__)}/tarballs/#{name}-0.1.tbz"
+        url "file://#{File.expand_path("..", __FILE__)}/tarballs/testball-0.1.tbz"
```

Comment:
If you're happy with my `/^testball/` suggestion, this would of course need to stay modified.